cost/azure/savings_plan/utilization/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v0.2.0
```

Comment:
Why 0.2.0 for an initial release?

Reply:
Bryan had initially made some changes while this policy template was in Draft that brought it to v0.1.9 - I was continuing based on this premise.

I am happy to change to 0.1.0 if @bryankaraffa is okay with that

Reply:
That's what I'd recommend. No real need to track internal working versions.